Test_Automation_Framework/EPAM_TestAutomation_Tests/Advansed_EpamSiteNavigateTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Epam_TestAutomation_BusinessLogic.PageObjects.Pages;
+using Epam_TestAutomation_Core.Browser;
+using Epam_TestAutomation_Core.Elements;
+using NUnit.Framework;
+using OpenQA.Selenium;
+
+namespace Epam_TestAutomation_Tests
+{
+    public class Advansed_EpamSiteNavigateTests : BaseTest
+    {
+        private SearchResultsPage _searchResultsPage;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _searchResultsPage = new SearchResultsPage();
+        }
+
+        [Test]
+        public void CheckTheOpeningOfTheLinkInTheCareerTabTest()
+        {
+            var linkJob = "https://www.epam.com/careers/job-listings";
+            var career = _searchResultsPage.CareerButton;
+            BrowserFactory.Browser.Action.MoveToElement(career.OriginalWebElement);
+            BrowserFactory.Browser.Action.Perform();
+            _searchResultsPage.JobListingsButton.Click();
+
+            Assert.That(BrowserFactory.Browser.GetUrl, Is.EqualTo(linkJob), "Incorrect url is present!");
+        }
+
+        [Test]
+        public void CheckListOfLanguagesTest()
+        {
+            var expectedlistOfLanguages = new List<string> {
+                "Global (English)", 
+                "Hungary (English)",
+                "СНГ (Русский)", 
+                "Česká Republika (Čeština)", 
+                "India (English)", 
+                "Україна (Українська)",
+                "Czech Republic (English)", 
+                "日本 (日本語)", 
+                "中国 (中文)", 
+                "DACH (Deutsch)", 
+                "Polska (Polski)"};
+            _searchResultsPage.LanguagesButton.Click();
+            var actualListOfLanguages = BrowserFactory.Browser.Waiters().
```

Comment:
move webElement to separate page object (Epam_TestAutomation_BusinessLogic.PageObjects.Pages)